.travis.yml

```diff
@@ -179,7 +142,7 @@ jobs:
 
     - name: "Deploy packages to integration test stack"
       stage: deployment test setup
-      install: true
+      install: /bin/true
```

Comment:
What is the meaning of this change?

Reply:
I took another look at the Travis docs, and this is a mistake.  I'll fix it.